app/models/form_profiles/va_21p530.rb

```diff
@@ -17,7 +17,11 @@ def prefill
     if @contact_information&.address&.country.present?
       @contact_information.address.country = convert_to_iso2(@contact_information.address.country)
     end
-    @military_information = initialize_military_information
+    ## We believe this is not being used. If you search form_profile_spec for "v21_p_530_expected"
+    ##  you'll see that there is no military information that is expected.
+    ## We found this while looking for other instances of initialize_military_information.
```

Comment:
No need to make this a comment, just remove it. You have the git history to tell you where to put it back in if you need it

Reply:
oops! Thank you for the reminder!